src/data/actions/courseInfo.js

```diff
@@ -154,7 +154,15 @@ function createCourseRun(courseUuid, courseRunData) {
         return dispatch(push(`/courses/${courseUuid}`));
       })
       .catch((error) => {
-        dispatch(createCourseRunFail(['Course Run create failed, please try again or contact support.'].concat(getErrorMessages(error))));
+        let errorList;
+        if (error.response.status === 504) {
```

Comment:
Might be worth confirming there's a response object here?

Reply:
Done.